admin-export.json

```diff
@@ -0,0 +1 @@
les bleus lui servent un gobelet de champagne. Tous en cercle, un verre \u00e0 la main, le t\u00e9l\u00e9phone dans l'autre, aucun ne d\u00e9colle de son \u00e9cran. Immobiles et silencieux, ils se livrent une guerre sans merci \u00e0 l'int\u00e9rieur du jeu, Yoyo attaquant, les bleus d\u00e9fendant. Les verts sont-ils l'\u00e9quipe de tyrans qu'on d\u00e9nonce ? \u00ab<em> On dit la m\u00eame chose des bleus ! <\/em>\u00bb<em>,<\/em> r\u00e9pond-il sans s'interrompre. \u00ab\u00a0<em>C'est quand m\u00eame un affrontement. Tout d\u00e9pend de la fa\u00e7on dont tu joues : en mode guerrier ou en <a class=\"lien_interne rub\" href=\"http:\/\/www.lemonde.fr\/m-mode\/\" title=\"Toute l\u2019actualit\u00e9 mode\">mode<\/a> amical <\/em>\u00bb<em>,<\/em> poursuit-il, sans pr\u00e9ciser laquelle des deux il a choisi d'adopter. Avant de dispara\u00eetre en courant.<\/p>\n<p>La pause d\u00e9jeuner touche \u00e0 sa fin. Le groupe commence \u00e0 se disperser, les quelques bleus restants cherchent \u00e0 r\u00e9parer les d\u00e9g\u00e2ts commis par Yoyo, avant de retourner \u00e0 leur quotidien. Pour quelques heures seulement.<\/p>","created_at":"2017-04-13T12:52:02+0200","updated_at":"2017-04-13T12:52:11+0200","annotations":[],"mimetype":"text\/html","language":"fr","reading_time":11,"domain_name":"www.lemonde.fr","preview_picture":"http:\/\/s1.lemde.fr\/image\/2015\/03\/27\/644x322\/4602939_3_ff8d_le-jeu-ingress-se-deroule-dans-le-monde-reel_08ebfe71833c30ac63929a410efb04e2.jpg","http_status":"200"},{"is_archived":0,"is_starred":1,"tags":["new tag 58ef585550a87","tag1","tag2"],"id":2,"title":"New awesome title++","url":"http:\/\/0.0.0.0\/entry2","content":"This is my content \/o\/","created_at":"2017-04-13T12:51:10+0200","updated_at":"2017-04-13T12:52:13+0200","annotations":[{"text":"This is my 2nd annotation \/o\/","quote":"content"}],"mimetype":"text\/html","language":"fr","reading_time":1,"domain_name":"domain.io"},{"is_archived":0,"is_starred":0,"tags":["foo bar","bar"],"id":4,"title":"test title entry4","url":"http:\/\/0.0.0.0\/entry4","content":"This is my content \/o\/","created_at":"2017-04-13T12:51:10+0200","updated_at":"2017-04-13T12:51:10+0200","annotations":[],"mimetype":"text\/html","language":"en","reading_time":12,"domain_name":"domain.io"},{"is_archived":0,"is_starred":1,"tags":[],"id":5,"title":"test title entry5","url":"http:\/\/0.0.0.0\/entry5","content":"This is my content \/o\/","created_at":"2017-04-13T12:51:10+0200","updated_at":"2017-04-13T12:51:10+0200","annotations":[],"mimetype":"text\/html","language":"fr","reading_time":12,"domain_name":"domain.io","preview_picture":"http:\/\/0.0.0.0\/image.jpg"},{"is_archived":1,"is_starred":0,"tags":["bar"],"id":6,"title":"test title entry6++","url":"http:\/\/0.0.0.0\/entry6","content":"This is my content \/o\/","created_at":"2017-04-13T12:51:10+0200","updated_at":"2017-04-13T12:52:07+0200","annotations":[],"mimetype":"text\/html","language":"de","reading_time":12,"domain_name":"domain.io"}]
```

Comment:
It's the generated json from console tests which outputs in the main directory. We should add them to `.gitignore`.